libs/java/dal/db/scripts/close-staging-repo.sh

```diff
@@ -7,5 +7,5 @@ if [ ! -z "$varStagingRepoId" ]
 then
   cd dal-db
   export MAVEN_OPTS="--add-opens=java.base/java.util=ALL-UNNAMED --add-opens=java.base/java.lang.reflect=ALL-UNNAMED --add-opens=java.base/java.text=ALL-UNNAMED --add-opens=java.desktop/java.awt.font=ALL-UNNAMED"
-  mvn nexus-staging:close -DstagingRepositoryId="cabcgovtno-$varStagingRepoId"
+  sudo env PATH="$PATH" mvn nexus-staging:close -DstagingRepositoryId="cabcgovtno-$varStagingRepoId" -s /home/vscode/.m2/settings.xml
```

Comment:
For some reason we need to use `sudo` to be able to `gpg` when the docker container is hosted on some operating systems (MacOS, WSL).